setup/git/010_TestSetup.rb

```diff
@@ -22,8 +22,8 @@
       #           host_path: ~/puppet
       #           container_path: /build/puppet
       #
-      if agent[:mount_folders]
-        mount = agent[:mount_folders][repository[:name]]
+      if host[:mount_folders]
+        mount = host[:mount_folders][repository[:name]]
```

Comment:
it turns out, `agent` and `agents` is equivalent if you don't explicitly set `agent` to something. Who knew! The loop that this exists in we refer to the individual servers as `host`, not `agent` as I had previously, and incorrectly, assumed

Reply:
I knew this sounded familiar: https://tickets.puppetlabs.com/browse/BKR-558